Decisions.Equifax/PrequalificationOfOne/PrequalificationOfOneSteps.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Decisions.Equifax.ConsumerCreditReport.Request;
+using Decisions.Equifax.PreQualificationOfOne.Response;
+using DecisionsFramework;
+using DecisionsFramework.Design.Flow;
+using DecisionsFramework.ServiceLayer;
+
+namespace Decisions.Equifax.PrequalificationOfOne
+{
+    [AutoRegisterMethodsOnClass(true, "Integration/Equifax/Pre-qualification of One")]
+    public class PreQualificationOfOneSteps 
+    {
+        /// <summary>
+        /// Step: Get Pre-qualification of One (Integration/Equifax/Consumer Credit Report/Pre-qualification of One)
+        /// </summary>
+        public static PreQualificationOfOneResponse GetPrequalificationOfOne(ConsumerCreditReportRequest request)
+        {
+            string scope = ModuleSettingsAccessor<EquifaxSettings>.GetSettings().EquifaxPreQualificationOfOneScope;
+            string requestUrl = ModuleSettingsAccessor<EquifaxSettings>.GetSettings().EquifaxPreQualificationOfOneEndpoint;
```

Comment:
Same comment as other step about creating a variable for the settings so we're not fetching twice.